core/tests/protractor/accessibility.js

```diff
@@ -103,7 +103,7 @@ describe('screenreader and keyboard user accessibility features', function() {
 
   it('should skip to the main content element', async function() {
     await libraryPage.get();
-    await browser.actions().sendKeys(protractor.Key.TAB).perform();
+    await browser.actions().sendKeys('s').perform();
```

Comment:
Changed this to 's' because `Skip Link Button` at library page shows up with pressing this key and not `TAB`, please take a look at `bindLibraryPageShortcuts` function in `KeyboardShortcutService`

Reply:
@jimbyo Just one thinjg I'd like to point!